app/views/provider/admin/dashboards/_services_navigation.html.slim

```diff
@@ -2,7 +2,7 @@ nav.DashboardNavigation.DashboardNavigation--services
   ul.DashboardNavigation-list
     - if can? :manage, :partners
 
-      - if current_user.accessible_services.size > 1
```

Comment:
For what it's worth, it was clearer this way to me

Reply:
Then that means that the name of the method is not ideal @josemigallas so propose a more appropriate name 😄 but the fact of being in a method instead of repeating logic makes total sense to me.